javascript/javascript1/week1/task3.js

```diff
@@ -0,0 +1,37 @@
+console.log('Housey pricey!') // Housey pricey (A house price estimator)
+
+let peter = {
+    houseWidth: 8,
+    houseDepth: 10,
+    houseHight: 10,
+    gardenSizeInM2: 100,
+    houseCost: 2500000
+}
+
```

Comment:
Very nice usage of JS Object. you could use 'const' to declare the variable/object unless you want to re-assign the value of the variable.